code/game/machinery/computer/_computer.dm

```diff
@@ -9,11 +9,17 @@
 	max_integrity = 200
 	integrity_failure = 100
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 40, "acid" = 20)
+	var/obj/item/card/id/scan = FALSE
```

Comment:
don't even initialize this

Reply:
> This needs a lot of cleanup but these are some nice quality of life changes. If you have any questions, just ask in the relevant comment.

Thanks! I'm currently figuring out how to make all the copypasted insert/eject ID -code into as few procs as possible. I'll commit that and I'm pretty sure someone can point out a more elegant way of doing it.